### DISCUSSION TITLE
TeamSpeak3

### DISCUSSION THREAD

**Discussion by gameinside** [SmileYzn/PugMod-AMXX#32] [Ideas]
it would also be fun to add a plugin to the connection to the teamspeak3 server, https://forums.alliedmods.net/showthread.php?t=90352?t=90352 optimize it for PUG

**Comment by SmileYzn** [comment]
This plugin works with client protocols (like steam://connect/xxx.xxx.xxx.xxx:27015).   
Actually we have ReHLDS dead talk support for live matches.

And now in CS 1.6 (2013 update like suggested on topic) that is not supported to open client protocols to use it with applications in operating systems.

What is the main idea of that? Maybe can reach this in other way.

**Reply by gameinside** [reply]
it's a shame, but the idea is not bad

**Reply by SmileYzn** [reply]
You can just put in motd.txt the address of voice servers and more things.
People just need to open it.

Maybe try to put links in an motd html page to users interact with it
